tapdb/assets_store.go

```diff
@@ -202,6 +205,14 @@ type ActiveAssetsStore interface {
 	// serialized outpoint.
 	DeleteManagedUTXO(ctx context.Context, outpoint []byte) error
 
+	// LeaseManagedUTXO leases a managed UTXO identified by the passed
+	// serialized outpoint.
+	LeaseManagedUTXO(ctx context.Context, arg LeaseManagedUTXO) error
```

Comment:
Same type and method name here? Maybe for the type name: `InsertLeasedUTXO`?

Reply:
Right, makes sense. I went with `UpdateUTXOLease` and `DeleteUTXOLease` since we'll also want to be able to extend the lease in the future.